halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/openstack/OpenstackEditAccountCommand.java

```diff
@@ -75,20 +87,26 @@ protected String getProviderName() {
   private Boolean insecure;
 
   @Parameter(
-      names = "--user-data-file",
+      names = "--heat-template-location",
       converter = PathExpandingConverter.class,
-      description = OpenstackCommandProperties.USER_DATA_FILE_DESCRIPTION
+      description = OpenstackCommandProperties.HEAT_TEMPLATE_LOCATION_DESCRIPTION
   )
-  private String userDataFile;
+  private String heatTemplateLocation;
 
   @Parameter(
-      names = "--remove-user-data-file",
```

Comment:
Yep, Lars explained a difference between these removes and others in halyard,only removes that edit a list should be implemented the halyard way is to set the variable to nothing